### ISSUE TITLE
cleaner.rb:65: warning: Using the last argument as keyword parameters is deprecated

### ISSUE THREAD

**Issue by jrgns** [bugsnag/bugsnag-ruby#580]
### Description
Run on Ruby 2.7

### Issue
Deprecation warnings are sent to stdout:

```
/vendor/bundle/ruby/2.7.0/gems/bugsnag-6.12.2/lib/bugsnag/cleaner.rb:65: warning: Using the last argument as keyword parameters is deprecated
```

### Environment

Library versions: bugsnag 6.12.2
Ruby: 2.7

- OS / OS version: Ubuntu
- production

See https://piechowski.io/post/last-arg-keyword-deprecated-ruby-2-7/

**Comment by mattdyoung** [comment]
Thanks @jrgns. We'll look at addressing this in a future release.

**Comment by mattdyoung** [comment]
Fixed in [v6.13.0](https://github.com/bugsnag/bugsnag-ruby/releases/tag/v6.13.0)
